<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>189</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Зоопарк</string>
  </property>
  <widget class="DragDropLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>39</y>
     <width>52</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Медведь</string>
   </property>
  </widget>
  <widget class="DragDropLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>39</y>
     <width>32</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Волк</string>
   </property>
  </widget>
  <widget class="DragDropLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>39</y>
     <width>45</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Лисица</string>
   </property>
  </widget>
  <widget class="DragDropLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>39</y>
     <width>32</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Заяц</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragDropLabel</class>
   <extends>QLabel</extends>
   <header>dragdroplabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
